src/Twig/Extension/MediaExtension.php

```diff
@@ -207,11 +205,11 @@ private function getMedia($media)
         }
 
         if (!$media instanceof MediaInterface) {
-            return false;
+            return null;
```

Comment:
Hmm this could be a be a be break. Did you double check the usage in this class?

Reply:
yes